sdk/src/dlob/DLOBSubscriber.ts

```diff
@@ -104,17 +104,13 @@ export class DLOBSubscriber {
 		}
 
 		let oraclePriceData;
-		let fallbackBid;
-		let fallbackAsk;
 		const isPerp = isVariant(marketType, 'perp');
 		if (isPerp) {
 			const perpMarketAccount =
 				this.driftClient.getPerpMarketAccount(marketIndex);
 			oraclePriceData = this.driftClient.getOracleDataForPerpMarket(
 				perpMarketAccount.marketIndex
 			);
-			fallbackBid = calculateBidPrice(perpMarketAccount, oraclePriceData);
-			fallbackAsk = calculateAskPrice(perpMarketAccount, oraclePriceData);
```

Comment:
can you fix the yarn lint / prettier errors (example is importing fcn but removing this line)